packages/ai-ide/src/browser/ai-configuration/mcp-configuration-widget.tsx

```diff
@@ -211,29 +212,41 @@ export class AIMCPConfigurationWidget extends ReactWidget {
                         </span>
                     </div>
                     <div style={{ flexGrow: 1 }}>
-                        <span className="mcp-section-label">Tools: </span>
+                        <span className="mcp-section-label">{nls.localize('theia/ai/mcpConfiguration/tools', 'Tools: ')}</span>
                     </div>
                     <div style={{ display: 'flex', gap: '4px' }}>
                         {this.renderButton(
                             <i className="codicon codicon-versions"></i>,
-                            'Copy all (multiple lines prompttemplate)',
+                            nls.localize('theia/ai/mcpConfiguration/copyAllList', 'Copy all (list of all tools)'),
                             e => {
                                 e.stopPropagation();
                                 if (server.tools) {
                                     const toolNames = server.tools.map(tool => `~{mcp_${server.name}_${tool.name}}`).join('\n');
                                     navigator.clipboard.writeText(toolNames);
-                                    this.messageService.info('Copied all tools to clipboard (multiple lines prompttemplate)');
+                                    this.messageService.info(nls.localize('theia/ai/mcpConfiguration/copiedAllList', 'Copied all tools to clipboard (list of all tools)'));
                                 }
                             },
                             'mcp-copy-tool-button'
                         )}
                         {this.renderButton(
                             <i className="codicon codicon-copy"></i>,
-                            'Copy all (single line prompttemplate)',
+                            nls.localize('theia/ai/mcpConfiguration/copyAllSingle', 'Copy all (single prompt fragment with all tools)'),
                             e => {
                                 e.stopPropagation();
-                                navigator.clipboard.writeText(`~{${this.mcpFrontendService.getPromptTemplateId(server.name)}}`);
-                                this.messageService.info('Copied all tools to clipboard (single line prompttemplate)');
+                                navigator.clipboard.writeText(`#${PROMPT_VARIABLE.name}:${this.mcpFrontendService.getPromptTemplateId(server.name)}`);
+                                this.messageService.info(nls.localize('theia/ai/mcpConfiguration/copiedAllSingle', 'Copied all tools to clipboard (single prompt fragment with \
+                                    all tools)'));
+                            },
+                            'mcp-copy-tool-button'
+                        )}
+                        {this.renderButton(
```

Comment:
Nitpick: Can we move this button before the other one (one to the left)? Then the `Copy` icons are aligned on the right side